runtime/lua/vim/fs.lua

```diff
@@ -2,7 +2,9 @@ local uv = vim.uv
 
 local M = {}
 
-local iswin = uv.os_uname().sysname == 'Windows_NT'
+-- Can't use `has('win32')` because the `nvim -ll` test runner doesn't support `vim.fn` yet.
+local sysname = uv.os_uname().sysname:lower()
+local iswin = sysname:find('windows') or sysname:find('mingw')
```

Comment:
We should be using has() here, but can't yet because `fs_spec.lua` runs its (many) tests in the `nvim -ll` test runner, which doesn't support `vim.fn` yet.